packages/patternfly-3/patternfly-react/src/components/DateTimePicker/DateComponents/Header.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Icon } from '../../../index';
+import { addDays, getWeekStart } from './helpers';
+import times from 'lodash/times';
+
+class Header extends React.Component {
+  getWeekArray = () => {
+    const { weekStartsOn } = this.props;
+    const weekStart = getWeekStart(new Date());
+    const dayFormat =
+      Intl.DateTimeFormat(this.props.locale, { weekday: 'short' }).format(weekStart).length > 3 ? 'narrow' : 'short';
+    return times(7, i =>
+      Intl.DateTimeFormat(this.props.locale, { weekday: dayFormat })
+        .format(addDays(weekStart, (i + weekStartsOn) % 7))
+        .slice(0, 2)
+    );
+  };
+  render() {
+    const { getNextMonth, getPrevMonth, toggleDateView } = this.props;
+    const date = new Date(this.props.date);
+    const month = Intl.DateTimeFormat(this.props.locale, { month: 'long' }).format(date);
+    const year = date.getFullYear();
+    const daysOfTheWeek = this.getWeekArray();
+    return (
+      <thead>
+        <tr>
+          <th className="prev" onClick={getPrevMonth}>
+            <Icon type="fa" name="angle-left" />
+          </th>
+          <th className="picker-switch" colSpan="5" onClick={() => toggleDateView('Y')}>
+            {month} {year}
+          </th>
+          <th className="next" onClick={getNextMonth}>
+            <Icon type="fa" name="angle-right" />
+          </th>
+        </tr>
+        <tr>
+          {daysOfTheWeek.map((day, idx) => (
+            <th key={idx} className="dow">
```

Comment:
Is this a bootstrap class? It doesn't appear to have any values. I can see it being useful to applications to give access to the styling, but `dow` isn't all that intuitive to me.

Reply:
it was in the [patternfly code](https://www.patternfly.org/pattern-library/forms-and-controls/date-and-time/#code), should I remove it?

Reply:
It's ok, it at least gives a hook. If anyone is moving from pf -> pf-react, they might expect this class to still be on the items.